.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: node_js
+
+node_js:
+  - 7
+  - 6
+
+cache: yarn
+
+notifications:
+  email: false
```

Comment:
Why turn off email?

I think email is useful. We should keep the option to the user.